src/_c_internal_utils.c

```diff
@@ -124,6 +124,47 @@ mpl_SetForegroundWindow(PyObject* module, PyObject *arg)
 #endif
 }
 
+static PyObject*
+mpl_SetDpiAwareness(PyObject* module)
```

Comment:
Would vaguely prefer e.g. mpl_SetProcessDpiAwareness_max (and likewise on the Python side) to stick to the underlying api names.